webpack.login.js

```diff
@@ -0,0 +1,108 @@
+const _ = require( 'lodash' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+const UglifyJSPlugin = require( 'uglifyjs-webpack-plugin' );
+const webpack = require( 'webpack' );
+
+// load in the babel config from babelrc and disable commonjs transform
+// this enables static analysis from webpack including treeshaking
+// also disable add-module-exports. TODO: remove add-module-exports from babelrc. requires fixing tests
+const babelConfig = JSON.parse( fs.readFileSync( './.babelrc', { encoding: 'utf8' } ) );
+const babelPresetEnv = _.find( babelConfig.presets, preset => preset[ 0 ] === 'env' );
+babelPresetEnv[ 1 ].modules = false;
+_.remove( babelConfig.plugins, elem => elem === 'add-module-exports' );
+
+const cacheIdentifier = require( './server/bundler/babel/babel-loader-cache-identifier' );
+
+const babelLoader = {
+	loader: 'babel-loader',
+	options: Object.assign( {}, babelConfig, {
+		babelrc: false,
+		cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
+		cacheIdentifier: cacheIdentifier,
+		plugins: [
+			...babelConfig.plugins,
+			'/Users/yury/Automattic/wp-calypso/inline-imports.js',
```

Comment:
I'm sorry @yurynix, I lost your password and can no longer access your laptop. As such, I can't access this plugin, and the following error is thrown when loading the `Login` page:

<img width="557" alt="screenshot" src="https://user-images.githubusercontent.com/594356/35684197-7678d7f0-0766-11e8-940b-7ad144d510ad.png">